chpl/chpl-resources/src/main/resources/jobs.xml

```diff
@@ -597,12 +321,48 @@
                 </entry>
             </job-data-map>
         </job>
-        <!-- Pending Change Request data emailer by ONC-ACB-->
+
+        <!-- Job fired when developer is split -->
         <job>
-            <name>ONC-ACB Pending Change Request Report</name>
+            <name>splitDeveloperJob</name>
+            <group>chplBackgroundJobs</group>
+            <description>Complete the work of splitting a developer and send an email notification to indicate if the operation was successful.</description>
+            <job-class>gov.healthit.chpl.scheduler.job.SplitDeveloperJob</job-class>
+            <durability>true</durability>
+            <recover>false</recover>
+            <job-data-map>
+                <entry>
+                    <key>authorities</key>
+                    <value>ROLE_ADMIN;ROLE_ONC;ROLE_ACB</value>
+                </entry>
+                <entry>
+                    <key>email</key>
+                    <value></value>
+                </entry>
+            </job-data-map>
+        </job>
+
+        <!-- Real World Testing Eligibility Job -->
+        <job>
+            <name>realWorldTestingEligibilityJob</name>
+            <group>systemJobs</group>
+            <description>For 2015 listings, determines if the listing is eligible for Real World Testing for the following year.</description>
+            <job-class>gov.healthit.chpl.scheduler.job.RealWorldTestingEligibilityJob</job-class>
+            <durability>true</durability>
+            <recover>false</recover>
+            <job-data-map>
+                <entry>
+                    <key>authorities</key>
+                    <value>ROLE_ADMIN;ROLE_ONC</value>
+                </entry>
+            </job-data-map>
+        </job>
+
```

Comment:
Can we add a comment above this job that marks the transition to ONC-ACB specific jobs? Something like the ` <!-- Real World Testing Eligibility Job -->` things above?